tools/prow-auto-bumper/file.go

```diff
@@ -56,57 +56,57 @@ func (pv *PRVersions) updateAllTags(content []byte, imageFilter *regexp.Regexp)
 			res += pv.images[image][iv].newVersion
 			msg += fmt.Sprintf("\nImage: %s\nOld Tag: %s\nNew Tag: %s", image, tag, pv.images[image][iv].newVersion)
 		} else {
-			errMsg := fmt.Sprintf("Cannot find version for image: '%s:%s'.\n", image, tag)
-			log.Println(errMsg)
-			errMsgs = append(errMsgs, errMsg)
+			tmp := fmt.Sprintf("there's no new version for image %s, keeping version: '%s:%s'.\n", image, image, tag)
```

Comment:
```suggestion
			tmp := fmt.Sprintf("There's no new version for image %s, keeping version: '%s:%s'.\n", image, image, tag)
```
Nit, capitalize since it's a sentence.